README.md

```diff
@@ -1,12 +1,11 @@
+# Tidepool Developer Portal
 
 This is a Tidepool microsite for helping to guide and manage the open source nature of our work.
 
-This is a GitHub Pages site, and the deployed site can be found [here](http://tidepool-org.github.io).
+This is a GitHub Pages site, and the deployed site can be found [here](http://developer.tidepool.io/).
 
-If you want to run it locally, you should have a Ruby installation working.
+If you want to run it locally, you should have a Ruby installation working and follow [GitHub's instructions for installing Jekyll](https://help.github.com/articles/using-jekyll-with-pages/#installing-jekyll).
 
-Then do ```gem install bundle``` from within this repository.
```

Comment:
Does the Jekyll install above negate the need to install bundler and all the rest of the gems?